ui/dashboards/src/components/Panel/Panel.tsx

```diff
@@ -100,6 +111,7 @@ export const Panel = memo(function Panel(props: PanelProps) {
         id={headerId}
         title={definition.spec.display.name}
         description={definition.spec.display.description}
+        readHandlers={readHandlers}
```

Comment:
somewhere I see view, somewhere I see full, here it says read.

I´d replace all wording by a common one no? What can we say fullView? focus? extended? maximized?

Reply:
Ok nevermind I think I follow what you're trying to do here.